app/assets/stylesheets/layout.scss

```diff
@@ -1,4 +1,56 @@
 // All the custom class styles
+.noborder {
+  border: 0;
+  border-radius: 0;
+  padding: 1.25rem;
+}
+
+.astrikstyle {
+  color: $danger;
+  font-size: 10px;
+}
+
+.aligncenter {
```

Comment:
There is already a built in bootstrap class to align center, what does this do ?